src/overlays/actors/ovl_En_Test5/z_en_test5.h

```diff
@@ -3,16 +3,20 @@
 
 #include "global.h"
 
-struct EnTest5;
+struct EnTest5; // SpringWaterModifier
 
 typedef void (*EnTest5ActionFunc)(struct EnTest5*, GlobalContext*);
 
 typedef struct EnTest5 {
     /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0x14];
-    /* 0x0158 */ EnTest5ActionFunc actionFunc;
+    /* 0x0144 */ Vec3f minPos;
+    /* 0x0150 */ f32 xLength;
+    /* 0x0154 */ f32 zLength;
+    /* 0x0158 */ EnTest5ActionFunc actionFunc;    
 } EnTest5; // size = 0x15C
 
 extern const ActorInit En_Test5_InitVars;
 
+#define IS_VARIANT_HOT_SPRING(this) (this->actor.params != 0)
```

Comment:
```suggestion
#define ENTEST5_IS_HOT_SPRING(this) (this->actor.params != 0)
```
Prefixing macros with the overlay name prevents possible conflicts. Also variant is unnecessary imo.

Reply:
Small thing, but is `EN_{ACTOR}` or `EN{ACTOR}` preferred for naming? I see both used throughout the codebase, and my current PRs use the underscore, but I'll switch them over if no underscore is preferred.

Reply:
I generally see/use `EN{ACTOR}`, though it is probably not that big a deal.